keras_nlp/models/bert/bert_preprocessing.py

```diff
@@ -18,119 +18,105 @@
 from tensorflow import keras
 
 from keras_nlp.layers.multi_segment_packer import MultiSegmentPacker
-from keras_nlp.models.bert.bert_presets import vocabularies
+from keras_nlp.models.bert.bert_models import Bert
 from keras_nlp.tokenizers.word_piece_tokenizer import WordPieceTokenizer
 
 
-def _handle_pretrained_tokenizer_arguments(vocabulary, lowercase):
-    """Look up pretrained defaults for tokenizer arguments.
+def download_vocabulary(preset, url, hash):
+    """Download vocabulary pretaining to preset."""
```

Comment:
"pertaining" (but you might want to use "associated with" to increase Levenshtein distance from "pretrained")